website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

```diff
@@ -1,17 +1,28 @@
 ---
-title: "Upgrade Core version in Cloud"
-id: "upgrade-core-in-cloud"
+title: "Upgrade dbt version in Cloud"
+id: "upgrade-dbt-version-in-cloud"
 ---
 
 In dbt Cloud, both [jobs](/docs/deploy/jobs) and [environments](/docs/dbt-cloud-environments) are configured to use a specific version of dbt Core. The version can be upgraded at any time.
 
 ## Environments
 
-Navigate to the settings page of an environment, then click **edit**. Click the **dbt Version** dropdown bar and make your selection. From this list, you can select an available version of Core to associate with this environment.
+Navigate to the settings page of an environment, then click **Edit**. Click the **dbt version** dropdown bar and make your selection. You can select a previous release of dbt Core or [Keep on latest version](#keep-on-latest-version). Be sure to save your changes before navigating away.
 
-<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/Environment-settings.png" title="Settings of a dbt Cloud environment"/>
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/example-environment-settings.png" width="90%" title="Example environment settings in dbt Cloud"/>
+
+### Keep on latest version <Lifecycle status='beta' />
+
+By choosing to **Keep on latest version**, you always get the latest fixes and early access to new functionality for your dbt project. dbt Labs will handle upgrades for you, as part of testing and redeploying the dbt Cloud SaaS application.
```

Comment:
Is the "early" part the selling point we want to make? With it, it could read like we're just forcing beta features into their environment, but alternatively, maybe it reads a little more like "you'll always have access to the newest functionality without having to select a version."

```suggestion
By choosing to **Keep on latest version**, you always get the latest fixes and immediate access to new features and functionality for your dbt project. dbt Labs will handle upgrades for you as part of testing and redeploying the dbt Cloud SaaS application.
```